src/stores/map.ts

```diff
@@ -38,7 +38,7 @@ export const useMapStore = defineStore("map", () => {
           buildingAbbreviation: space.building.abbreviation,
           buildingNumber: space.building.number,
           buildingOccupancy: space.building.occupancy,
-          isOpen: spaceIsOpen(now, space.openingHours),
+          isOpen: true,
```

Comment:
have another look at this one :)

Reply:
Removed in f0e9a47 it looks like this state is unused? The filter and such still work without it.